From de90b8ac92910a6208f0e35a0051898e85bd78d4 Mon Sep 17 00:00:00 2001
From: Jason <jasonanderson@uchicago.edu>
Date: Thu, 10 Jan 2019 14:36:37 -0600
Subject: [PATCH 35/52] Trying more syntax tricks

---
 Jenkinsfile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/Jenkinsfile b/Jenkinsfile
index 5dbb1e0..2c742c7 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -2,9 +2,7 @@ pipeline {
   agent any
  
   options {
-    copyArtifactPermissionProperty {
-      projectNames('horizon-container')
-    }
+    copyArtifactPermissionProperty(projectNames: ['horizon-container'])
   }
 
   stages {
-- 
1.8.3.1

